### ISSUE TITLE
Tool name prefixing

### ISSUE THREAD

**Issue by leenaars** [arpa2/tlspool#51]
As a service to packagers, would it be possible to prefix the names used for the tools with "tlspool-"?

So tlspool-have_db instead  of have_db, etc? Other applications have databases too ;)

**Comment by vanrein** [comment]
Hey,

> As a service to packagers, would it be possible to prefix the names
> used for the tools with "tlspool-"?

Yes this makes sense and should be done < 1.0.  It's also in the interest of people installing >1 package, I suppose :)

> So tlspool-have_db instead of have_db, etc? Other applications have
> databases too ;)

Same.  Although the _ should be renamed to - and I should think if the names also make sense on command lines.  This particular routine, by the way, may only be interesting during installation.  Then again... once we add a multi-user mode we should also consider users who setup a directory and then need an empty database.  Any work done now to bootstrap an installation would then be useful to bootstrap a user.

-Rick

**Comment by vanrein** [comment]
Resolved in ba7826c9a6169369d4acbb502b5d8d407313ed25

*(This is bound to have quite some impact on places where we already use the TLS Pool!)*

## New installation:

```
/usr/local/sbin/tlspool-daemon
/usr/include/tlspool/commands.h
/usr/include/tlspool/async.h
/usr/include/tlspool/uthash.h
/usr/include/tlspool/internal.h
/usr/include/tlspool/starttls.h
/usr/include/tlspool/commands.h.new
/usr/include/tlspool/utlist.h
/usr/local/lib/libtlspool.so
/usr/local/lib/libtlspool.a
/usr/local/lib/libtlspool_async.so
/usr/local/lib/libtlspool_async.a
/usr/local/sbin/tlspool-configvar
/usr/local/sbin/tlspool-disclose-get
/usr/local/sbin/tlspool-localid-get
/usr/local/sbin/tlspool-trust-get
/usr/local/sbin/tlspool-db-have
/usr/local/sbin/tlspool-localid-select
/usr/local/sbin/tlspool-pin-entry
/usr/local/sbin/tlspool-ping
/usr/local/sbin/tlspool-ping-async
/usr/local/sbin/tlspool-disclose-set
/usr/local/sbin/tlspool-localid-set
/usr/local/sbin/tlspool-trust-set
/usr/local/sbin/tlspool-test-client
/usr/local/sbin/tlspool-test-server
/usr/local/sbin/pgp11-genkey
/usr/local/sbin/tlstunnel
/usr/local/share/steamworks/pulleyback/pulleyback_tlspool.so
/usr/local/share/doc/tlspool/socketprotocol.rst
/usr/local/share/doc/tlspool/databases.rst
/usr/local/share/doc/tlspool/identities.rst
/usr/local/share/doc/tlspool/pkcs11.rst
/usr/local/share/doc/tlspool/validation.md
/usr/local/share/doc/tlspool/startxxx.md
/usr/local/share/doc/tlspool/p2p-tls.md
/usr/local/share/doc/tlspool/ircproxy-explained.md
/usr/local/share/doc/tlspool/localid-selection.md
/usr/local/share/doc/tlspool/anonymising-precursor.md
/usr/local/share/doc/tlspool/COPYRIGHT.MD
/usr/local/share/doc/tlspool/LICENSE-DAEMON.MD
/usr/local/share/doc/tlspool/LICENSE-DOCS.MD
/usr/local/share/doc/tlspool/LICENSE-USERSPACE.MD
/usr/local/share/man/man3/tlspool_socket.3
/usr/local/share/man/man3/tlspool_starttls.3
/usr/local/share/man/man3/tlspool_ping.3
/usr/local/share/man/man3/tlspool_prng.3
/usr/local/share/man/man3/tlspool_pin_service.3
/usr/local/share/man/man3/tlspool_localid_service.3
/usr/local/share/man/man3/tlspool_control_detach.3
/usr/local/share/man/man3/tlspool_control_reattach.3
/usr/local/share/man/man3/tlspool_configvar.3
/usr/local/share/man/man8/tlspool-daemon.8
/usr/local/share/man/man8/tlstunnel.8
```

## Old installation:

```
/usr/local/sbin/tlspool
/usr/include/tlspool/commands.h
/usr/include/tlspool/async.h
/usr/include/tlspool/uthash.h
/usr/include/tlspool/internal.h
/usr/include/tlspool/starttls.h
/usr/include/tlspool/commands.h.new
/usr/include/tlspool/utlist.h
/usr/local/lib/libtlspool.so
/usr/local/lib/libtlspool.a
/usr/local/lib/libtlspool_async.so
/usr/local/lib/libtlspool_async.a
/usr/local/sbin/configvar
/usr/local/sbin/get_disclose
/usr/local/sbin/get_localid
/usr/local/sbin/get_trust
/usr/local/sbin/have_db
/usr/local/sbin/lidsel
/usr/local/sbin/pgp11_genkey
/usr/local/sbin/pinentry
/usr/local/sbin/pingpool
/usr/local/sbin/set_disclose
/usr/local/sbin/set_localid
/usr/local/sbin/set_trust
/usr/local/sbin/testcli
/usr/local/sbin/testsrv
/usr/local/sbin/tlstunnel
/usr/local/share/steamworks/pulleyback/pulleyback_tlspool.so
/usr/local/share/doc/tlspool/socketprotocol.rst
/usr/local/share/doc/tlspool/databases.rst
/usr/local/share/doc/tlspool/identities.rst
/usr/local/share/doc/tlspool/pkcs11.rst
/usr/local/share/doc/tlspool/validation.md
/usr/local/share/doc/tlspool/startxxx.md
/usr/local/share/doc/tlspool/p2p-tls.md
/usr/local/share/doc/tlspool/ircproxy-explained.md
/usr/local/share/doc/tlspool/localid-selection.md
/usr/local/share/doc/tlspool/anonymising-precursor.md
/usr/local/share/doc/tlspool/COPYRIGHT.MD
/usr/local/share/doc/tlspool/LICENSE-DAEMON.MD
/usr/local/share/doc/tlspool/LICENSE-DOCS.MD
/usr/local/share/doc/tlspool/LICENSE-USERSPACE.MD
/usr/local/share/man/man3/tlspool_socket.3
/usr/local/share/man/man3/tlspool_starttls.3
/usr/local/share/man/man3/tlspool_ping.3
/usr/local/share/man/man3/tlspool_prng.3
/usr/local/share/man/man3/tlspool_pin_service.3
/usr/local/share/man/man3/tlspool_localid_service.3
/usr/local/share/man/man3/tlspool_control_detach.3
/usr/local/share/man/man3/tlspool_control_reattach.3
/usr/local/share/man/man3/tlspool_configvar.3
/usr/local/share/man/man8/tlspool.8
/usr/local/share/man/man8/tlstunnel.8
```
